src/app/components/login/login.component.ts

```diff
@@ -22,14 +26,5 @@ export class LoginComponent {
   errorMessage: string | null = null;
 
   onSubmit(): void {
-    const { email, password } = this.form.getRawValue();
-    this.authService.login(email, password).subscribe(
-      () => {
-        this.router.navigate(['/']);
-      },
-      (error) => {
-        this.errorMessage = error.message;
-      }
-    );
```

Comment:
I assume since you removed everything here this function is empty now? just make sure it gets used in the future or to remove it if it won't be